backend/user_management/minio/utils.py

```
@@ -3,6 +1,18 @@
+from fastapi import UploadFile, HTTPException
 
 from fastapi import UploadFile
 from user_management.minio.config import MINIO_CLIENT, BUCKET_NAME
+from minio.error import S3Error
+from urllib.parse import urlparse
+from typing import List
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)